CoreFoundation/Parsing.subproj/CFPropertyList.c

```diff
@@ -2178,7 +2178,8 @@ static Boolean parseXMLElement(_CFXMLPlistParseInfo *pInfo, Boolean *isKey, CFTy
             if (pInfo->skip) {
                 *out = NULL;
             } else {
-                *out = CFRetain(kCFBooleanTrue);
+                SInt32 val = 1;
+                *out = CFNumberCreate(kCFAllocatorSystemDefault, kCFNumberSInt8Type, (const void *)&val);
```

Comment:
This change doesn't make a lot of sense to me. `CFPropertyList` is the same on both platforms, so if there is a behavior difference here it is due to bridging - not the creation of booleans in this file.

Also, returning the singleton for booleans is a huge performance advantage vs. creating a new one each time.